docs/core/tools/dotnet.md

```diff
@@ -68,7 +68,7 @@ Prints out detailed information about the CLI tooling and the environment, such
 
  Rolls forward on no candidate shared framework.
 
-`-v|--verbose`
+`-v|--verbosity`
 
 Enables verbose output.
```

Comment:
This should specify the options for verbosity, which I think are: `q[uiet], m[inimal], n[ormal], d[etailed], and diag[nostic]`